resume.html

```diff
@@ -1,10 +1,66 @@
 <!DOCTYPE html>
+
 <html>
+
   <head>
-    <title>Personal Website - Resume</title>
+    <title>Jack Carpenter's Personal - Resume</title>
     <link rel="stylesheet" type="text/css" href="css/style.css">
   </head>
+
   <body>
-    
-  </body>
+  
+	<img src="images/img_5404.jpeg">
+
+  <h1>RESUME</h1>
+  <h2>of</h2>
+  <h2>Jack Carpenter</h2>
+  <h3>Prologue</h3>
+
+  <h4>I am anticipating graduating from a Front End Web Development Course being held at General Assembly in New York City.  I have limited computer skills compared to most of the students in my class it seems, however I've got a giant heart and the will to continue learing this skill that others pick up so easily it seems.  I have always enjoyed holistic and spiritual objectives.  Love the outdoors.  I desire finding a revenue source that this new skill will allow me.</h4> 
+
+<h3>Revelent Experience</h3>
+
+<h4>August 2016</h4>
+	<p>FEWD Graduate FEWD Class 90</p>
+	
+
+<h3>Past Experience</h3>
+
+<h4>1996-Present</h4>	
```

Comment:
If you tab indent all these items to be even with there sibling <p> tags it will make the code easier to read.